curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-396-weak-goodstein-sequence.md

```diff
@@ -8,43 +8,51 @@ dashedName: problem-396-weak-goodstein-sequence
 
 # --description--
 
-For any positive integer n, the nth weak Goodstein sequence {g1, g2, g3, ...} is defined as:
+For any positive integer $n$, the $n$th weak Goodstein sequence $\\{g1, g2, g3, \ldots\\}$ is defined as:
 
-g1 = n
+- $g_1 = n$
+- for $k > 1$, $g_k$ is obtained by writing $g_{k - 1}$ in base $k$, interpreting it as a base $k + 1$ number, and subtracting 1.
 
-for k > 1, gk is obtained by writing gk-1 in base k, interpreting it as a base k + 1 number, and subtracting 1.
+The sequence terminates when $g_k$ becomes 0.
 
-The sequence terminates when gk becomes 0.
+For example, the $6$th weak Goodstein sequence is $\\{6, 11, 17, 25, \ldots\\}$:
 
-For example, the 6th weak Goodstein sequence is {6, 11, 17, 25, ...}: g1 = 6. g2 = 11 since 6 = 1102, 1103 = 12, and 12 - 1 = 11. g3 = 17 since 11 = 1023, 1024 = 18, and 18 - 1 = 17. g4 = 25 since 17 = 1014, 1015 = 26, and 26 - 1 = 25.
+- $g_1 = 6$.
+- $g_2 = 11$ since $6 = 1102$, $1103 = 12$, and $12 - 1 = 11$.
+- $g_3 = 17$ since $11 = 1023$, $1024 = 18$, and $18 - 1 = 17$.
+- $g_4 = 25$ since $17 = 1014$, $1015 = 26$, and $26 - 1 = 25$.
```

Comment:
```suggestion
- $g_2 = 11$ since $6 = 110_2$, $110_3 = 12$, and $12 - 1 = 11$.
- $g_3 = 17$ since $11 = 102_3$, $102_4 = 18$, and $18 - 1 = 17$.
- $g_4 = 25$ since $17 = 101_4$, $101_5 = 26$, and $26 - 1 = 25$.
```